rest/data/host_create_test.go

```diff
@@ -403,4 +406,7 @@ buildvariants:
 	assert.Equal("docker.io/library/hello-world", h.DockerOptions.Image)
 	assert.Equal("echo hi", h.DockerOptions.Command)
 	assert.Equal(distro.DockerImageBuildTypePull, h.DockerOptions.Method)
+	assert.Len(h.DockerOptions.EnvironmentVars, 2)
+	assert.Equal(h.DockerOptions.EnvironmentVars[0], "banana=yellow")
```

Comment:
There is no guarantee of the order when you iterate over a map (see "Iteration Order" in https://blog.golang.org/go-maps-in-action). One way to check the environment variables would be to use variables or another map:
```
banana := false
apple := false
for k, v := range h.DockerOptions.EnvironmentVars {
   if k == "apple" {
    apple = true
  }
  if k == "banana" {
    banana = true
  }
}
assert.True(apple)
assert.True(banana)
```